diffexpr/py_deseq.py

```diff
@@ -65,7 +68,8 @@ class py_DESeq2:
 
     """
 
-    def __init__(self, count_matrix, design_matrix, design_formula, gene_column="id"):
+    def __init__(self, count_matrix, design_matrix, design_formula, gene_column="id", threads=4):
```

Comment:
My understanding is that setting `threads=4` here will cause the default function to run in parallel with 4 threads. Maybe a better solution is to have the default setting run serially with `threads=1`. Then the user can update the threads param if they are interested in parallelization.

Reply:
maybe also implement a simple check that the threads parameter passed by the user is an integer.
`if not isinstance(threads, int):
    raise ValueError("threads must be an integer")`

Reply:
good suggestions!!